src/scenario/message_media_text_blob.c

```diff
@@ -106,7 +124,9 @@ static text_blob_string_t *create_text_blob(int length)
         log_error("This will overfill the message_media_text_blob. The game will now crash.", 0, 0);
     }
 
-    message_media_text_blob.text_entries[index].id = index;
+    // Increment before use, even if 0, since we do not want to use 0.
```

Comment:
There should be a better way to do this rather than relying on a comment.

Reply:
Removed the comment. It was meant to explain why we increment before use and not after, but is clearly misleading.